testonly/loglb/main.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/google/trillian/monitoring"
 	"github.com/google/trillian/util"
 
+	"errors"
```

Comment:
nit: Ditto.

Maybe reorder so we have 2 import blocks: language and non-language?